cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/ClientNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+
+package mil.army.usace.hec.cwms.radar.client;
+
+import java.io.IOException;
+
+public class ClientNotFoundException extends IOException {
```

Comment:
Naming for this exception?

Reply:
yeah this doesn't make sense. Renamed to ServerNotFoundException